code/modules/awaymissions/mission_code/ruins/moonoutpost19.dm

```diff
@@ -0,0 +1,37 @@
+
+/obj/item/paper/fluff/ruins/moonoutpost19/boreraccountant
+	name = "confusing note"
+	info = "I had a weird dream recently about something small catching me off-guard, from under my desk where they've hidden. <br><br>And now something in my mind whispers to me about 'tax norms', how it is implemented and various terms i have never heard before. <br><br> A note to self: don't push yourself too hard over small handicaps. It obviously makes you go insane, in a scary way too... And ask your therapist for extra session regarding to this matter."
+
+/obj/item/paper/fluff/ruins/moonoutpost19/log1
+	name = "Chamber #1 Log"
+	info = "<hr><b>Data Log #82<br><br> Subject: Yes<br><br> Details:</b><br><br> I fricking hate our so called 'Senior Researcher' Mr. Wood. Someone has to show him how we handle the pricks whose nose in the air in this line of business.<br><br> Therefore i have developed a gas cannon. What this device does is simply compressing air and releasing it in a specific direction -in this case it would be the airlock of this very chamber- at a funny velocity. He checks the containtment chambers daily, all i have to do is keep this beauty locked and loaded until he opens the airlock. Afterall the only difference between messing around and science is writing it down. I bet you won't be so senior about this."
+
+/obj/item/paper/fluff/ruins/moonoutpost19/log2
+	name = "Chamber #2 Log"
+	info = "<hr><b>Data Log #74<br><br> Subject: Bizzare Bread<br><br> Details:</b><br><br> So, a few weeks ago our custodian Bob noticed loaf of breads appearing around the outpost. At first we didn't believe him of course, there is no reasonable way to put this situation in this colorful existence... Until i get the honor to actually witnessing it. Someone or something was teleporting bread directly to this location.<br><br> Despite my best efforts to isolate outpost from this specific bluespace frequency the results have met with failure. This was silly and i couldn't be arsed so i told Bob to do his damn job. A few days later something extraordinary happened, the object has shown the sign of a sort of decay process which resembling more of a mutation. An object was experiencing mutation as if it was a living organism!<br><br> Investigations are still in progress, we'll be keeping all specimens locked in this chamber."
+
+/obj/item/paper/fluff/ruins/moonoutpost19/truth
+	name = "unsent note"
+	info = "As you have ordered, i managed to insert the device in one of the containtment shield generators and cut one or two wires feeding the emergency systems. Remote signal is clear and in range, waiting for next step... <br><br>This will cost many lives but whatever make you guys satisfied and lessens my debt."
```

Comment:
```suggestion
	info = "As you have ordered, i managed to insert the device in one of the containment shield generators and cut one or two wires feeding the emergency systems. Remote signal is clear and in range, waiting for next step... <br><br>This will cost many lives but whatever make you guys satisfied and lessens my debt."
```